### ISSUE TITLE
Support of Health Connect Google app

### ISSUE THREAD

**Issue by Atreyu-94** [jonasoreland/runnerup#1149]
This app allows synchronization between different health data among different apps, according to the description: ‘Android’s Health Connect offers you a simple way to share data between health, fitness and wellness apps without compromising your privacy.’

https://play.google.com/store/apps/details?id=com.google.android.apps.healthdata

**Comment by gerhardol** [comment]
It does not seem like it is designed for getting data realtime or to handle activities (history).
Will take a closer look later.

**Comment by Atreyu-94** [comment]
Indeed, it is at a very early stage I guess, it is on beta yet. But it could be a good thing later :)

**Comment by matejdro** [comment]
It does handle activities: https://developer.android.com/guide/health-and-fitness/health-connect/common-workflows/work-with-sessions#exercise_sessions

But you are correct that it is not real time (although you could theoretically feed workout real time, most apps seem to just sync it after the fact).

**Comment by andreas-breitbach** [comment]
Hey there,
I'd like to point out that integrating with Health Connect would also enable the app [PAIesque](https://codeberg.org/ojrandom/paiesque) to work with data generated by RunnerUP.
This way, you'd be able to train with RunnerUP and check the recordings against data calculated by PAIesque to improve  your training. Health Connect also works on de-googled GrapheneOS phones.
Kind regards,
Andreas
